### DISCUSSION TITLE
Only execute formulas with none empty cells

### DISCUSSION THREAD

**Discussion by julianaiglsdorfer** [handsontable/hyperformula#901] [Feature requests]
Hello,

is there a way to calculate a formula only if every cell it contains is also filled?
For example, if I want to calculate the sum of 2 values, that only a sum is formed when both cells are filled, and not one of them is empty. 
I've created a fiddle to demonstrate it -> https://jsfiddle.net/yj23tf9a/1/.

Thanks in advance!

**Comment by bardek8** [comment]
In order to change it globally for all cells in HF, you should modify interpreter not to coerce null values.

But maybe a simpler solution will work for you, namely modify formulas so that they first check if the used cells are not empty?
For example, in your fiddle in cell A3 you can have:
`=IF(AND(NOT(ISBLANK(A1)),NOT(ISBLANK(A2))),A1+A2,"")`

**Reply by julianaiglsdorfer** [reply]
Do you have an example how i can change this interpreter?

Since I am designing a business application, customizing the formula is very cumbersome and not really doable. 
If here is thought to introduce an option with which this behavior can be set please let me know.

**Reply by sequba** [reply]
> If here is thought to introduce an option with which this behavior can be set please let me know.

Please, create an issue (a feature request) and describe the intended behaviour. We will consider adding it to the scope of the next release.

**Reply by sequba** [reply]
Issue: #950
